### PR TITLE
bugfix: fix prometheus fail to return seata metrics data when using Nacos

### PR DESCRIPTION
**Pull request by slievrly**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did

fix prometheus fail to return seata metrics data when using Nacos

### CODE REVIEW

File: common/src/main/java/org/apache/seata/common/DefaultValues.java [hunk 1]
```diff
@@ -259,7 +259,7 @@ public interface DefaultValues {
     /**
      * the const DEFAULT_METRICS_ENABLED
      */
-    boolean DEFAULT_METRICS_ENABLED = false;
+    boolean DEFAULT_METRICS_ENABLED = true;
```
Comment (funky-eyes) [inline common/src/main/java/org/apache/seata/common/DefaultValues.java:262]: Why is the default value changed to true?

File: script/config-center/config.txt [hunk 2]
```diff
@@ -164,7 +164,7 @@ server.raft.sync=true
 
 
 #Metrics configuration, only for the server
-metrics.enabled=false
+metrics.enabled=true
```
Comment (funky-eyes) [inline script/config-center/config.txt:167]: Why is the default value changed to true?

File: common/src/main/java/org/apache/seata/common/DefaultValues.java (hunk 1 above)
Comment (slievrly) [reply common/src/main/java/org/apache/seata/common/DefaultValues.java:262]: Monitoring data should be exposed from the kernel layer by default, and whether to display it is controlled by whether the service layer is connected to the corresponding monitoring system.

**Comment by xjlgod** [comment]
LGTM.I have tested by some examples.

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**